### ISSUE TITLE
Faster center transition for dark-centered diverging maps

### ISSUE THREAD

**Issue by drscotthawley** [1313e/CMasher#50]
**Describe the colors you would like**
I want  'bwr' only with black in the middle, i.e..  I want "dark mode bwr": Blue on one side, red on the other, black in the middle, but for a fairly narrow range in the middle. 

I notice that all the light-centered diverging maps appear as if there is a "rapid" transition around the central value, whereas for the darker maps the central dark colors cover a might wider proportion of the colormap. 

My collaborator hates all the white in this image and asked for a "dark mode", so I'm working on that.
<img width="1006" alt="Screen Shot 2022-12-05 at 5 50 58 PM" src="https://user-images.githubusercontent.com/13925685/205770595-b1da3789-a857-439b-84ca-5521611e6b76.png">

'redshift' has too much of the middle of the color range as near-black:
<img width="971" alt="Screen Shot 2022-12-05 at 5 53 18 PM" src="https://user-images.githubusercontent.com/13925685/205771015-67dda102-399b-4f1b-8d6a-e0cb02f0e9b9.png">
I want the black to cover a more narrow range in the middle. 

 'guppy' seems to provide the most rapid transition of the dark-centered diverging colormaps, but I don't want purple in the middle. Most of my plot ends up looking purple (see screenshot below), whereas I'd rather have it mostly dark. 
<img width="1108" alt="Screen Shot 2022-12-05 at 5 49 46 PM" src="https://user-images.githubusercontent.com/13925685/205770329-b58ac0c9-bd19-4747-81d8-1c0e68099ba8.png">

**Describe the colormap type**

diverging

**Describe any extra properties**

I'm not sure how one gets colorblind-friendliness in a diverging map, but if possible, that'd be great! 

**Is your colormap request related to a specific application? Please describe.**

Sure, see [`tokens_spectrogram_image()`](https://drscotthawley.github.io/aeiou/viz.html#tokens_spectrogram_image) in my [`aeiou`](https://github.com/drscotthawley/aeiou) library.  
I want to easily see at a glance the distribution of positive, negative, and "small" (near zero) values in a 2D array. 

**Additional context**

screenshots above

**Comment by 1313e** [comment]
@drscotthawley,

hmm, to me, the second plot you posted with `wildfire` looks really good actually, when I compare it to the `bwr` plot above it.
I can see all elements in both plots.
The reason it becomes harder to see the red/blue parts that are very close to 0 (on the far right of your plots for example), is because they should be.
They differ very little from 0, and thus should be near-black.

So, I have a return question first: Are ALL non-zero values in this plot important to clearly visualize?
Basically, any value that is not zero should be clearly distinct from actual zero?
If so, I could make a different colormap or a function (that CMasher could benefit from), that combines two sequential (10, 90) colormaps with black in the center.
Your smallest non-zero values would then become 10% lightness, which is visually distinct from black (0 lightness).
